packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-suggestions.tsx

```diff
@@ -0,0 +1,258 @@
+import {Component, h, Prop, State, Element} from '@stencil/core';
+import {
+  InitializableComponent,
+  InitializeBindings,
+  Bindings,
+  BindStateToController,
+} from '../../utils/initialization-utils';
+import ArrowRight from '../../images/arrow-right.svg';
+import ArrowDown from '../../images/arrow-down.svg';
+import {
+  buildInteractiveResult,
+  buildSmartSnippetQuestionsList,
+  SmartSnippetQuestionsList,
+  SmartSnippetQuestionsListState,
+  SmartSnippetRelatedQuestion,
+} from '@coveo/headless';
+import {Hidden} from '../common/hidden';
+import {Heading} from '../common/heading';
+import {LinkWithResultAnalytics} from '../result-link/result-link';
+import {Button} from '../common/button';
+import {randomID} from '../../utils/utils';
+
+/**
+ * The `atomic-smart-snippet-suggestions-suggestions` component displays an accordion of questions related to the query with their corresponding answers.
+ *
+ * You can style the snippets by inserting a template element like this:
+ * ```html
+ * <atomic-smart-snippet-suggestions>
+ *   <template>
+ *     <style>
+ *       b {
+ *         color: blue;
+ *       }
+ *     </style>
+ *   </template>
+ * </atomic-smart-snippet-suggestions>
+ * ```
+ *
+ * @part container - The wrapper with a border around the entire component.
+ * @part heading - The heading above the list of related question.
+ * @part questions - The list of related questions.
+ * @part question-answer-expanded - An expanded related question.
+ * @part question-answer-collapsed - A collapsed related question.
+ * @part question-expanded-button - The button to collapse a related question.
+ * @part question-collapsed-button - The button to expand a related question.
+ * @part question-expanded-icon - The caret of an expanded related question.
+ * @part question-collapsed-icon - The caret of a collapsed related question.
+ * @part question-expanded-text - The title of an expanded related question.
+ * @part question-collapsed-text - The title of a collapsed related question.
+ * @part answer-and-source - The wrapper around the answer and source of a related question.
+ * @part answer - The container displaying the full document excerpt of a related question's answer.
+ * @part footer - The wrapper around the source of a related question's answer.
+ * @part source-url - The URL to the document a related question's answer is extracted from.
+ * @part source-title - The title of the document a related question's answer is extracted from.
+ *
+ * @internal
```

Comment:
I added `@internal` until the bug with multiple snippets expanding/collapsed at once is fixed in Headless.